m4/pbs_decl_socklen_t.m4

```diff
@@ -45,9 +45,10 @@ AC_DEFUN([PBS_AC_DECL_SOCKLEN_T],
 #include <netdb.h>
 ], [
 #ifdef  __hpux
-  NO socklen_t in HPUX
+#error NO socklen_t in HPUX
 #else
-  socklen_t       len;
+  socklen_t       len = 0;
+  len++;
```

Comment:
It's being incremented so that the compiler doesn't throw a warning that the variable is unused. When compiling with '-Wall -Werror' it would cause the compilation to fail, which we don't want.

Reply:
If the variable is unused, then why not remove it?

Reply:
This isn't code that gets compiled into any PBS Pro binary. It's code for autoconf to test whether socklen_t should be defined as int or the native socklen_t. It produced the wrong result when -Werror was added to CFLAGS because what had been a warning was now an error. We're just silencing the compiler so it produces the result we want in this case. The code is correct as it stands.